arlo_server/routes.py

```diff
@@ -42,6 +42,8 @@
 from util.jurisdiction_bulk_update import bulk_update_jurisdictions
 from util.process_file import process_file
 
+import arlo_server.all_routes.election.__element__
```

Comment:
I'm down with putting routes into a subdirectory, but this nested structure feels a bit more confusing to me given the size/structure of the API. The pattern I already started was to put each logical set of routes (loosely defined) in its own file within `arlo_server`, and then import them in `__init__.py`.

I think this kind of thing can get confusing when there are multiple organization systems in place, so I prefer to either do small changes that don't make a radically new scheme or migrate everything to the new scheme at once.

Could you follow the pattern I established for now and we can come back and reorg and break up routes.py in a separate PR?

For this route specifically, I think it would group well with `/election/new` and `/election/audit/reset` since those are the C and the D that accompany these endpoints (the R and U) of the election CRUD methods.

Reply:
I didn't see that you'd already established a pattern here. I'll take a look.

Reply:
I moved to `arlo_server/election_settings.py`.